lib/AST/ASTBridging.cpp

```diff
@@ -1236,6 +1328,15 @@ BridgedTypeRepr BridgedDictionaryTypeRepr_createParsed(
                          SourceRange{lSquareLoc, rSquareLoc});
 }
 
+BridgedTypeRepr
+BridgedInverseTypeRepr_createParsed(BridgedASTContext cContext,
```

Comment:
I noticed all the current `BridgedXXXTyprRepr_createParsed()` returns `BridgedTypeRepr` instead of the concrete wrapper types. Since we haven't implemented checked down casting, I think it'd be better to keep the sub type as long as possible (and that's consistent with Decl/Stmt/Expr). WDYT @hamishknight ?

Reply:
👍 Will do in a followup